eap74-rhel8-payg-multivm/src/main/bicep/mainTemplate.bicep

```diff
@@ -211,8 +201,8 @@ var linuxConfiguration = {
 var name_vmAcceptTerms = format('vmAcceptTerms{0}', guidValue)
 var imageReference = {
   publisher: 'RedHat'
-  offer: ((jdkVersion == 'openjdk8') ? 'rh-jboss-eap' : 'RHEL')
-  sku: ((jdkVersion == 'openjdk8') ? 'rh-jboss-eap74-rhel8' : '8_6')
+  offer: 'rh-jboss-eap'
+  sku: (jdkVersion == 'openjdk8') ? 'rh-jboss-eap74-jdk8-rhel8' : (jdkVersion == 'openjdk11') ? 'rh-jboss-eap74-jdk11-rhel8' : (jdkVersion == 'openjdk17') ? 'rh-jboss-eap74-jdk17-rhel8' :  null
```

Comment:
I do understand there is no way to do this other than `? :` but it sure is not easy to read.